packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -111,17 +124,22 @@ export function DAppsExplorerScreen() {
           }
         })
         result.applications.forEach((app: any) => {
-          categoriesById[app.categoryId].dapps.push({
-            id: app.id,
-            categoryId: app.categoryId,
-            name: app.name,
-            iconUrl: app.logoUrl,
-            description: app.description,
-            dappUrl: app.url.replace('{{address}}', address ?? ''),
-          })
+          categoriesById[app.categoryId].dapps.push(mapDappFields(app, address ?? ''))
         })
 
-        return Object.values(categoriesById)
+        const featured = result.featured ? mapDappFields(result.featured, address ?? '') : undefined
```

Comment:
Wondering if the featured info returned by the API should be only the dapp ID, and then we should get the info from the list of applications. Otherwise, we might have duplicated information.